src/pages/AddPhrases.js

```diff
@@ -16,7 +29,48 @@ export default function AddPhrases({ phrases, handlePhraseSubmit, showMessage, h
       {showMessage ? (
         <ModalPhraseAddedMessage message="Dein Kind wurde erfolgreich hinzugefügt! Füge nun einen Spruch hinzu!" />
       ) : null}
-      <PhraseForm phrases={phrases} handlePhraseSubmit={handlePhraseSubmit} />
+      {kidsData.length === 0 ? (
+        <StyledLinkEmptyKidsData to="/createkids" onClick={() => handleImageUrl('')}>
+          <AddKidsIcon fill="#19337a" height="40px" width="40px" />
+          <span>Füge ein Kind hinzu!</span>
+        </StyledLinkEmptyKidsData>
+      ) : (
+        <PhraseForm
+          imageUrl={imageUrl}
+          kidsData={kidsData}
+          loadingProcess={loadingProcess}
+          phrases={phrases}
+          isPreviewLoading={isPreviewLoading}
+          handleImageUrl={handleImageUrl}
+          handlePhraseSubmit={handlePhraseSubmit}
+          handleImageUploadInPhraseForm={handleImageUploadInPhraseForm}
+        />
+      )}
     </>
   );
 }
+
+const StyledLinkEmptyKidsData = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0.5rem 0;
+  gap: 10px;
+  color: #19337a;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  min-height: 250px;
+  min-width: 300px;
+  border: 1px dashed #19337a;
+  background-color: #f9f9f9;
+  border-radius: 10px;
+
+  span {
+    display: block;
+    font-size: 1.2rem;
+    color: #19337a;
```

Comment:
I would recommend you to define color variables

Reply:
hey chris!thx for reviewing! that´s a good idea 👍 I´ll do it when I´ll refactor the code